setup.py

```diff
@@ -68,7 +68,13 @@
 It is almost fully compliant with the Python database API version 2.0 also
 exposes the unique features of SQLite."""
 
-if sys.platform != "win32":
+try:
+    subprocess.check_output(["uname", "-a"])
```

Comment:
There is no specific reason to use check_output() vs check_call(). check_call() would actually be more preferable.

The "-a" argument also is excessive and can be removed.